lib/esignatur/client.rb

```diff
@@ -13,6 +12,10 @@ def orders
       Orders.new(api: api)
     end
 
+    def user(user_id, creator_id)
```

Comment:
I would suggest to add `users` method instead. Similar as with `orders`. This method does not look consistent with other parts

```ruby
    def users
      Users.new(api: api)
    end
    # usage: Client.new.users.find_by(user_id: user_id, creator_id: creator)
```